sdks/python/apache_beam/io/filebasedsink.py

```diff
@@ -65,9 +66,12 @@ def __init__(self,
                compression_type=CompressionTypes.AUTO):
     """
      Raises:
-      TypeError: if file path parameters are not a string or ValueProvider,
-                 or if compression_type is not member of CompressionTypes.
-      ValueError: if shard_name_template is not of expected format.
+      ~exceptions.TypeError: if file path parameters are not a :class:`str` or
```

Comment:
Why is syntax for classes alternating between `~exceptions.TypeError`  :class:`~apache_beam.options.value_provider.ValueProvider`